id-repository/id-repository-identity-service/src/main/java/io/mosip/idrepository/identity/validator/IdRequestValidator.java

```diff
@@ -180,38 +184,40 @@ public boolean supports(Class<?> clazz) {
 	 */
 	@Override
 	public void validate(@Nonnull Object target, Errors errors) {
+		boolean isDraftRequest = servletRequest.getRequestURI().contains("Draft");
 		if (target instanceof IdRequestDTO) {
 			IdRequestDTO request = (IdRequestDTO) target;
 
-			validateReqTime(request.getRequesttime(), errors);
+			if (!isDraftRequest)
+				validateReqTime(request.getRequesttime(), errors);
 
-			if (!errors.hasErrors()) {
+			if (!errors.hasErrors() && !isDraftRequest) {
 				validateVersion(request.getVersion(), errors);
 			}
 
-			if (!errors.hasErrors() && Objects.nonNull(request.getId())) {
+			if (!isDraftRequest && !errors.hasErrors() && Objects.nonNull(request.getId())) {
 				if (request.getId().equals(id.get(CREATE))) {
 					validateStatus(request.getRequest().getStatus(), errors, CREATE);
 					validateRequest(request.getRequest(), errors, CREATE);
 				} else if (request.getId().equals(id.get(UPDATE))) {
 					validateStatus(request.getRequest().getStatus(), errors, UPDATE);
 					validateRequest(request.getRequest(), errors, UPDATE);
 				}
-
-				validateRegId(request.getRequest().getRegistrationId(), errors);
 			}
+			if (servletRequest.getRequestURI().endsWith("updateDraft")) {
```

Comment:
Similar part assumption here. This will also break

Reply:
@ramesh-n Updated code with a safe approach